proposals/dao/v2Phase1.js

```diff
@@ -73,6 +76,8 @@ async function run(addresses, oldContracts, contracts, logging = false) {
   await ratioPCVController.withdrawRatio(oldContracts.dpiUniswapPCVDeposit.address, dpiUniswapPCVDeposit.address, '10000'); // move 100% of PCV from old -> new
 
   await core.allocateTribe(feiTribeLBPSwapper.address, `1000000${e18}`);
+
```

Comment:
This was causing an issue because the DPI bonding curve was looking at old DPI Uniswap deposit. Updating allocation